content/docs/setup/kubernetes/helm-install/index.md

```diff
@@ -34,6 +34,12 @@ plane and the sidecars for the Istio data plane.
    service objects.  For platforms lacking `LoadBalancer` support, install Istio with `NodePort` support
    instead with the flags `--set gateways.istio-ingressgateway.type=NodePort --set gateways.istio-egressgateway.type=NodePort`
    appended to the end of the Helm operation.
+   
+1. If you are running istio under docker for desktop's built-in kubernetes, you may need to increase docker's memory limit
+   under the *Advanced* pane of docker's preferences.  Pilot by default requests 2048Mi of memory, which is docker's
+   default limit.  Alternatively, you may reduce pilot's memory reservation by passing the helm argument
+   `--set pilot.resources.requests.memory="512Mi"`.  Otherwise pilot may refuse to start due to insufficient resources.
+   See [Installation Options](https://istio.io/docs/reference/config/installation-options) for more information.
```

Comment:
I don't see how a section about this would fit under the installation-options page.  My first inclination was to add this as a sub-page under https://istio.io/docs/setup/kubernetes/platform-setup/ like https://istio.io/docs/setup/kubernetes/platform-setup/minikube/, but I thought it didn't belong there because I'm talking about setting up istio itself, not getting the underlying kubernetes setup working.

I agree it would be nice to add a note about how to set the INGRESS_HOST, though.  I think it's just my machine's primary IP

I'm going to submit a separate pull request with notes about setting `global.proxy.includeIPRanges` egress rules to call external services directly for bare metal and docker-for-desktop environments after I do some more testing.

Reply:
Ah yes, sorry, https://istio.io/docs/setup/kubernetes/platform-setup/ is what I meant (I seem to have had a copy-paste error, when I pasted the link in my previous comment. I had the same thought as you, that much of what you are saying is about a specific issue one might have when installing the istio using helm, but I was thinking it would be best in the instructions that explain how to set up docker-desktop - those are the instructions for people that are planning to install Istio next. More generally, you're PR reminded me that we don't even have any specific setup/use instructions for docker-for-desktop, just minikube at the moment.

Reply:
So do you want me to create a docker-desktop section under platform-setup and move this?

On a broader level, should the documentation be organized differently so the platform-setup pages become more general platform-specific configuration pages covering setup and all the other nuances of a specific platform, from helm install tweaks to the details of egress filtering (see sections under https://istio.io/docs/tasks/traffic-management/egress/#set-the-value-of-global-proxy-includeipranges) and other sections with platform-specific notes should have links there when there are specialized instructions?  (I'm not offering to do that restructuring, just asking if that's the direction.)

Reply:
It would be good to get other opinions, but personally I think that's not preferable. I think the instructions for how to do specific tasks on various platforms belong with the task instructions, as they currently are. Things, like your PR, that talk about, how you should configure something in a platform (so it will work with Istio) are better in the platform setup instructions.